dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/DataOkHttpUploader.kt

```diff
@@ -30,6 +31,7 @@ internal abstract class DataOkHttpUploader(
 
         return try {
             val request = buildRequest(data)
+            sdkLogger.v("$TAG: ${String(data)}")
```

Comment:
I don't think you want this here...even though it will only display for us it will create that String object all the time which is very heavy.

Reply:
Good catch, it was for debug purposes